op-node/service.go

```diff
@@ -143,14 +148,29 @@ func NewL2SyncEndpointConfig(ctx *cli.Context) *node.L2SyncEndpointConfig {
 	}
 }
 
-func NewDriverConfig(ctx *cli.Context) *driver.Config {
+func NewConfigPersistence(ctx *cli.Context) node.ConfigPersistence {
+	stateFile := ctx.String(flags.RPCAdminPersistence.Name)
+	if stateFile == "" {
+		return node.DisabledConfigPersistence{}
+	}
+	return node.NewConfigPersistence(stateFile)
+}
+
+func NewDriverConfig(ctx *cli.Context, config node.ConfigPersistence) (*driver.Config, error) {
+	sequencerStopped := ctx.Bool(flags.SequencerStoppedFlag.Name)
+	if state, err := config.SequencerState(); err != nil {
+		return nil, err
+	} else if state != node.StateUnset {
+		sequencerStopped = state == node.StateStopped
+	}
```

Comment:
- It may be worth a warn-log when the CLI flag disagrees with the persisted state
- It may be worth an info-log when no persisted state was found